### PR TITLE
rebrand logo

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description
This is part 1 of PROD-2270, rebranding of the support tools. This PR is updating the logo in the header and in the favicon. Besides updating logo, the PR is add `request` as a dev dependency to avoid test failures. Some eslint fixes have been done as well.

### CODE REVIEW

**Comment by saadyousafarbi** [comment]
@DawoudSheraz are there any screenshots for this?

**Review by saadyousafarbi — APPROVED** [review]
Can you share some screenshots for visual clarity? otherwise, LGTM 👍

**Comment by DawoudSheraz** [comment]
@saadyousafarbi This is how the default/dev behavior would be:

Screenshot 2021-01-22 at 2 46 39 PM

I changed the URL explicitly to prod to show how it would look like:

Screenshot 2021-01-22 at 2 48 22 PM

**Comment by saadyousafarbi** [comment]
@DawoudSheraz looks good! :shipit:

File: .env.development [hunk 1]
```diff
@@ -10,6 +10,10 @@ LMS_BASE_URL='http://localhost:18000'
 DISCOVERY_API_BASE_URL='http://localhost:18381'
 LOGIN_URL='http://localhost:18000/login'
 LOGOUT_URL='http://localhost:18000/logout'
+LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
+LOGO_TRADEMARK_URL=https://edx-cdn.org/v3/default/logo-trademark.svg
```
Comment (asadazam93) [inline .env.development:14]: Are we using all of these variables?

Comment (DawoudSheraz) [reply .env.development:14]: Not for now. I added them based upon the template https://github.com/edx/frontend-template-application/blob/master/.env.development

**Review by asadazam93 — APPROVED** [review]
Looks good

**Pull request merged**